inlayHints.py

```diff
@@ -0,0 +1,104 @@
+from .protocol import InlayHint, InlayHintRequestParams, InlayHintResponse
+from html import escape as html_escape
+from LSP.plugin import Request
+from LSP.plugin import Session
+from LSP.plugin.core.protocol import Point
+from LSP.plugin.core.registry import windows
+from LSP.plugin.core.types import debounced
+from LSP.plugin.core.types import FEATURES_TIMEOUT
+from LSP.plugin.core.typing import List, Optional, Tuple
+from LSP.plugin.core.views import point_to_offset
+from LSP.plugin.core.views import text_document_identifier
+import sublime
+import sublime_plugin
+
+
+def inlay_hint_to_phantom(view: sublime.View, hint: InlayHint) -> sublime.Phantom:
+    html = """
+    <body id="lsp-typescript-inlay-hints">
+        <style>
+            #lsp-typescript-inlay-hints .inlay-hint {{
```

Comment:
Here, internally you can just use `.inline-hint` selector. The `id` is to be able to override that using external stylesheet.

And while that idea is good, ST has a bug that probably makes overrides not work anyway. But hopefully it will be fixed eventually.